### PR TITLE
fix authorization

### PR DESCRIPTION
**Pull request by rzhang74**
*Issue #, if available:*

*Description of changes:*
Infrastructure:
1. Adding the authorizer for all APIs 

Frontend:
1. Adding the authorization header

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: app/lib/medical-transcription-analysis-stack.ts [hunk 1]
```diff
@@ -498,11 +498,11 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
...
             apiResource.addMethod(method, undefined, {
-              authorizationType: apigateway.AuthorizationType.NONE//apigateway.AuthorizationType.COGNITO,
-              // authorizer: {
-              //   authorizerId: `${authorizer.ref}`
-              //   //`${theauthorizer.ref}`
-              // }
+              authorizationType: apigateway.AuthorizationType.COGNITO,
+              authorizer: {
+                authorizerId: `${authorizer.ref}`
+                //`${theauthorizer.ref}`
```
Comment (ShivaniMehendarge) [inline app/lib/medical-transcription-analysis-stack.ts]: Can we remove the comment?

**Review by ShivaniMehendarge — COMMENTED** [review]
Please remove the commented code. Rest LGTM

**Review by alexchirayath — APPROVED** [review]
Please remove commented code and merge in the changes

File: app/lib/medical-transcription-analysis-stack.ts (hunk 1 above)
Comment (rzhang74) [reply app/lib/medical-transcription-analysis-stack.ts]: Sure!

Comment (rzhang74) [reply app/lib/medical-transcription-analysis-stack.ts]: https://github.com/aws-samples/medical-transcription-analysis/blob/bb69041e36eed9769f2c959a29f1818e3a0983b5/app/lib/medical-transcription-analysis-stack.ts#L500-L501

**Pull request merged**